components/benefit_sponsors/app/models/benefit_sponsors/benefit_applications/benefit_sponsor_health_catalog_decorator.rb

```diff
@@ -50,11 +51,13 @@ def single_issuer_options
       end
 
       def metal_levels
-        plan_options[:metal_level].keys
+        return {} if plan_options[:metal_leve].nil?
+        return plan_options[:metal_level].keys
       end
 
       def single_product_options
         single_product_options_hash = {}
+        return single_product_options_hash if plan_options[:single_product].nil?
```

Comment:
I just added this as a hotfix to get past the errors it was causing when I click "Edit Plan Year"